CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8.c

```diff
@@ -19,10 +19,10 @@
 /* ----------------------------------------------------------------------
  * Project:      CMSIS NN Library
  * Title:        arm_depthwise_conv_s8.c
- * Description:	 s8 version of depthwise convolution.
+ * Description:  s8 version of depthwise convolution.
  *
- * $Date:        09. October 2020
- * $Revision:    V.2.0.1
+ * $Date:        11. May 2021
+ * $Revision:    V.2.0.2
```

Comment:
Just a minor comment..Since it is a feature addition, lets have it as 2.5.0.

Reply:
@felix-johnny it is already set to 2.5.0 in the second commit.